features/run/production.feature

```diff
@@ -12,15 +12,15 @@ Feature: running Exosphere applications
 
   Scenario: booting an exosphere application in development mode
     When starting "exo run" in my application directory
-    And it prints "nginx nginx online" in the terminal
+    And it prints "Attaching to nginx" in the terminal
```

Comment:
The output is actually now `nginx    | nginx online`, but for some reason execplus doesn't pick that up if I tell it to wait on that text. I had to wait on the line right before `nginx    | nginx online`, which is this.

Reply:
Running into a similar problem on `features/test/complex-test.feature` , on the step:
```
    And I eventually see:
      """
      1 scenario (1 passed)
      """
```
<img width="783" alt="screen shot 2017-10-27 at 1 16 04 pm" src="https://user-images.githubusercontent.com/7258240/32123454-1a189f3e-bb19-11e7-9229-2654e985e918.png">

Reply:
For the nginx part, your solution is fine. My PR addressed the complex-test feature